tests/functional/splinter_test.c

```diff
@@ -2383,9 +2383,6 @@ usage(const char *argv0)
       "\t%s --cache-per-table\n"
       "\t%s --parallel-perf --max-async-inflight [num] --num-pthreads [num] "
       "--lookup-positive-percent [num] --seed [num]\n"
-      "\t%s --num-normal-bg-threads (number of normal background threads)\n"
```

Comment:
Any reason why these lines were deleted? I thought these two params are supported when invoking this test-case.

I do see one case invoking this combination in `test.sh` on L555. So, they are valid outputs in `--help`:

```
555     run_with_timing "Functionality test, default key size, with background threads" \
556         "$BINDIR"/driver_test splinter_test --functionality 1000000 100 \
557                                             --num-normal-bg-threads 4 --num-memtable-bg-threads 2 \
558                                             --seed "$SEED"
```

Reply:
These have been replaced by the more general config.c system for specifying background threads.